common/djangoapps/lang_pref/views.py

```diff
@@ -0,0 +1,22 @@
+"""
+Language Preference Views
+"""
+import json
+from django.conf import settings
+from django.views.decorators.csrf import csrf_exempt
+from django.utils.translation import LANGUAGE_SESSION_KEY
+from lang_pref import LANGUAGE_KEY
+from django.http import HttpResponse
+
+
+@csrf_exempt
```

Comment:
@ziafazal I think, we can exempt this view from csrf, It is just updating the language session for unauthenticated users. I know that, we are sending csrf token in headers but it has been changed after user sign in. The csrf token is rotated when the user logs in. This was added as a security measure in Django 1.5.2 [release notes](https://docs.djangoproject.com/en/1.8/releases/1.5.2/).
please share your thoughts.